### ISSUE TITLE
keep URL when change window.location

### ISSUE THREAD

**Issue by samuweiss** [TrySound/skrollr-decks#24]
Hi everyone

I have a problem with the history.pushState(null, null, id) on Line 124. When I use it I don't get correct URL. With the following solution it work for me:
history.pushState(null, null,"//" +  window.location.hostname + window.location.pathname + id);

Thanks
Samuel

**Comment by TrySound** [comment]
Which url do you get?

**Comment by samuweiss** [comment]
i've get the base (from the head "base"-Tag) URL width the element ID.

http://example.ch/#c456
insed of
http://example.ch/animation/decks.html#c456

**Comment by TrySound** [comment]
@samuweiss Do you use pushState to set this url?

**Comment by samuweiss** [comment]
i don't think so. I thought this is the common behavier if a base tag is set.

http://stackoverflow.com/questions/8108836/make-anchor-links-refer-to-the-current-page-when-using-base

**Comment by TrySound** [comment]
@samuweiss What about `location.href.split("#")[0]` decision?

**Comment by samuweiss** [comment]
yes, I think that is the solution.

**Comment by TrySound** [comment]
@samuweiss Can you make a PR?
